src/utils/Processing.ts

```diff
@@ -57,13 +64,21 @@ export function ensureOutputDir(input: string, output: string, isIgPubContext: b
 }
 
 export function readConfig(input: string): Configuration {
-  const configPath = ensureConfiguration(input);
+  const configPath = ensureConfigurationFile(input);
+  let config: Configuration;
   if (configPath == null || !fs.existsSync(configPath)) {
-    logger.error('No config.yaml in FSH definition folder.');
+    config = loadConfigurationFromIgResource(input);
+  } else {
+    const configYaml = fs.readFileSync(configPath, 'utf8');
+    config = importConfiguration(configYaml, configPath);
+  }
+  if (!config) {
+    logger.error(
+      'No config.yaml in FSH definition folder, and no configuration could' +
+        ' be extracted from an ImplementationGuide JSON file.'
```

Comment:
I'm sorry to do this (I should have thought about it before), but should this actually say "ImplementationGuide resource" since it could be a JSON or XML?

Reply:
Yes it should!

Reply:
https://github.com/FHIR/sushi/pull/577/commits/f19af48d84428149513942705923060a5f5e10fc fixes this
.